packages/core/components/Puck/components/Preview/index.tsx

```diff
@@ -27,7 +37,7 @@ export const Preview = ({ id = "puck-preview" }: { id?: string }) => {
       }}
       style={{ zoom: disableZoom ? 1 : 0.75 }}
     >
-      <Page dispatch={dispatch} state={state} {...rootProps}>
+      <Page {...rootProps}>
```

Comment:
`dispatch` and `state` were kept here to retain backwards compatibility with [plugins from v0.12.0](https://puckeditor.com/v/0.12.0/docs/api-reference/plugins/functions#render-args), before it was changed in v0.13.0.

I think I'd rather retain these for now, but we should probably mark them as `// DEPRECATED` so we remember to pull them out during a breaking change release.